recipes/vtk-cdat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.10" %}
+{% set name = "vtk-cdat" %}
+
+package:
+    name: {{ name }}
+    version: 7.1.0.{{ version }}
+
+source:
+    git_rev: 1c03d1d03b1c12d99dd090ba4c5ca078e2d9612f
+    git_url: git://github.com/Kitware/VTK.git
```

Comment:
VTK doesn't put out nightly tarballs. This recipe enables us to use a VTK version that may be ahead of the official release.